tests/agent/mixins/agent_mq_mixin_test.py

```diff
@@ -44,6 +47,22 @@ async def testClient_whenMessageIsSent_processMessageIsCalled(mocker, mq_service
     assert stub.call_count == 1
 
 
+@pytest.mark.asyncio
+async def testConnection_whenConnectionException_reconnectIsCalled(mocker):
+    stub = mocker.stub(name="test1")
+    client = Agent.create(
+        stub, name="test1", keys=["d.#"], url="amqp://wrong:wrong@localhost:5672/"
+    )
+    task = asyncio.create_task(client.mq_init())
+
+    try:
+        await asyncio.wait_for(task, timeout=10)
+    except asyncio.TimeoutError:
+        pass
```

Comment:
shouldnt we handle this ?

Reply:
The main goal of this test is to make sure that reconnection is done; if it's not, this test will fail since we will get aiormq.exceptions. AMQPConnectionError error instead of asyncio.TimeoutError